exporters/otlp/otlp_metric_test.go

```diff
@@ -695,13 +700,78 @@ func TestResourceInstLibMetricGroupingExport(t *testing.T) {
 	)
 }
 
+func TestStatelessExportKind(t *testing.T) {
+	type kase struct {
```

Comment:
That reads like german Käse (which means cheese). `testcase` maybe?

Reply:
Done.